src/components/Icon/IconProject.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import StyleObjectPropType from '../../prop-types/style';
+import './Icon.scss';
+
+let lastInstanceId = 0;
+
+const propTypes = {
+    className: PropTypes.string,
+    style: StyleObjectPropType,
+};
+
+const defaultProps = {
+    className: null,
+    style: null,
+};
+
+const IconProject = ({
+    className,
+    style,
+}) => {
+    lastInstanceId += 1;
+    const iconTitleId = `icon-project-title-${lastInstanceId}`;
+
+    return (
+        <svg
+            className={cx('uir-icon uir-icon-project', className)}
+            style={style}
+            viewBox="-4 -1 24 24"
```

Comment:
Do we have to have negative values in the `viewBox`? Looks weird

Reply:
Might look weird but the icons are an odd shape, it's the best way I've found to get them to align correctly in the view box.  If we don't have the negative values it becomes a real pain to get the icons to align with text